src/promotion.jl

```diff
@@ -0,0 +1,71 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+const DefaultDatum = WGS84Latest
+
+const DefaultCRS = Behrmann
```

Comment:
```suggestion
# we promote to Behrmann by default because
# it preserves area near +/-30° latitude, but this
# may change in the future after we have better
# support for LatLon in downstream packages
const DefaultCRS = Behrmann

# we promote to WGS84Latest by default because
# it is the most widely used datum nowadays
const DefaultDatum = WGS84Latest
```